README.md

```diff
@@ -1,27 +1,27 @@
 # Mysterium Node - decentralized VPN built on blockchain
 
 [![Go Report Card](https://goreportcard.com/badge/github.com/mysteriumnetwork/node)](https://goreportcard.com/report/github.com/mysteriumnetwork/node)
-[![Build Status](https://travis-ci.org/mysteriumnetwork/node.svg?branch=master)](https://travis-ci.org/mysteriumnetwork/node)
+[![Build Status](https://travis-ci.com/mysteriumnetwork/node.svg?branch=master)](https://travis-ci.com/mysteriumnetwork/node)
```

Comment:
Hm, if I preview this file, it show "build | unknown"

Reply:
No builds for master were done right now on `travis-ci.com`. :)